tee/tee.go

```diff
@@ -135,38 +137,26 @@ func (s *Sig) UnmarshalJSON(data []byte) error {
 	return err
 }
 
-func (b Balance) MarshalJSON() ([]byte, error) {
-	aux := map[string]interface{}{
-		"epoch":   b.Epoch,
-		"account": b.Account,
-		"value":   b.Value.Text(10),
-	}
-	return json.Marshal(aux)
+func (s Amount) MarshalJSON() ([]byte, error) {
```

Comment:
Use `a` as var for all `Amount` methods and pointer.
```suggestion
func (s *Amount) MarshalJSON() ([]byte, error) {
```

Reply:
PS The marshall needs an object and not a pointer, it silently fails otherwise.